cluster-autoscaler/provisioningrequests/apis/autoscaling.x-k8s.io/v1beta1/types.go

```diff
@@ -139,13 +160,16 @@ type ProvisioningRequestStatus struct {
 	// +optional
 	Conditions []metav1.Condition `json:"conditions"`
 
-	// AdditionalStatus contains all other status values custom provisioning classes may require.
+	// Statuses contains all other status values custom provisioning classes may require.
 	//
 	// +optional
-	// +kubebuilder:validation:MinProperties=64
-	AdditionalStatus map[string]string `json:"additionalStatus"`
+	// +kubebuilder:validation:MaxProperties=64
+	Statuses map[string]Status `json:"statuses"`
```

Comment:
`.status.statuses` stutters a bit... trying to think of a non-"status" name for this... Details, ProvisioningClassDetails, maybe?

Reply:
ProvisioningClassDetails it is